transcom/mymove
# B 21580 unlock gov search queues

## [B-21580](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1024956)

## Summary
Original integration PR: https://github.com/transcom/mymove/pull/14111
KICKBACK PR: https://github.com/transcom/mymove/pull/14239

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Before running make/client servers change `FEATURE_FLAG_MOVE_LOCK=true` in your envrc
2. Run `direnv allow` and then you can start up your make/client servers
4. Login as 2 different QAE, GSR, CSR, or Headquarters office users in 2 different browsers.
5. Officer 1: Search for any existing move like DRTTFR
7. Officer 1: Click record for move to see move details page
9. Officer 1: Click Milmove button or naviagte away from move(signing out, closing out the page)
11. Officer 2: Search for the same move Officer 1 opened previously
12. Officer 2: Confirm there is no lock next to the record move code

### Backend


## Screenshots
**_Before_**
<img width="1682" alt="Screenshot 2024-11-04 at 11 53 14 PM" src="https://github.com/user-attachments/assets/648868a7-7510-42b3-817b-3bcb554db9e6">


**_After_**
<img width="1678" alt="Screenshot 2024-11-04 at 11 43 40 PM" src="https://github.com/user-attachments/assets/55ca3819-f354-496a-8573-2abcb4a20682">
